ipn/ipnstate/ipnstate.go

```diff
@@ -328,6 +328,14 @@ type PeerStatus struct {
 	KeyExpiry *time.Time `json:",omitempty"`
 
 	Location *tailcfg.Location `json:",omitempty"`
+
+	// The IPv4 address that this peer knows the current node as. It may be
+	// empty if the peer knows the current node by its native IPv4 address.
+	V4MasqAddr *netip.Addr `json:",omitempty"`
+
+	// The IPv6 address that this peer knows the current node as. It may be
+	// empty if the peer knows the current node by its native IPv6 address.
+	V6MasqAddr *netip.Addr `json:",omitempty"`
```

Comment:
we can drop the pointers here and use omitzero nowadays instead.


Also the field names should be in normal Go style starting with the subject, like all the fields above (https://go.dev/wiki/CodeReviewComments#comment-sentences)